### PR TITLE
Add language option to gatsby-source-wikipedia plugin

### PR DESCRIPTION
**Pull request by StevenHoppinger24K**
It might be usefull to also be able to search in different languages for wikipedia articles. 

Wikipedia is available in many languages so it does make sense to have a language option in the wikipedia plugin

### CODE REVIEW

**Comment by Steven24K** [comment]
@KyleAMathews I saw you wrote this plugin two years ago. What do you think about this language option?

File: packages/gatsby-source-wikipedia/src/fetch.js [hunk 1]
```diff
@@ -75,8 +75,11 @@ const search = ({ query, limit }) =>
...
-const getArticle = name =>
-  fetch(`https://en.m.wikipedia.org/wiki/${name}?action=render`)
+/**
+ * You can get wikipedia articles in different languages
+ */
+const getArticle = (name, lang = 'en') =>
+  fetch(`https://${lang}.m.wikipedia.org/wiki/${name}?action=render`)
     .then(res => res.text())
     .then(pageContent =>
       pageContent.replace(/\/\/en\.wikipedia\.org\/wiki\//g, `/wiki/`)
```
Comment (vladar) [inline packages/gatsby-source-wikipedia/src/fetch.js:85]: This `en` here probably also needs to be handled?

File: packages/gatsby-source-wikipedia/index.js [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+getArticle('Andries Tunru', 'nl').then(res => console.log(res))
+
+search({query: 'kaas', limit: 10, lang: 'nl'}).then(res => console.log(res))
+
+getMetaData('frommage', 'en')
+
+fetchNodesFromSearch({query: 'velo', lang: 'fr'})
+
+console.log('done.')
```
Comment (vladar) [inline packages/gatsby-source-wikipedia/index.js:13]: What's this? Probably a leftover after some testing? Please remove.

File: packages/gatsby-source-wikipedia/src/fetch.js (hunk 1 above)
Comment (StevenHoppinger24K) [reply packages/gatsby-source-wikipedia/src/fetch.js:85]: I don't really see what this replace is doing. I could not add the `lang` variable in that regex. But maybe if I just write it as a string.

File: packages/gatsby-source-wikipedia/index.js (hunk 2 above)
Comment (StevenHoppinger24K) [reply packages/gatsby-source-wikipedia/index.js:13]: This can be removed.

**Comment by LekoArts** [comment]
Hi,

this PR became stale so I'm closing it for now. There's also a similar PR (https://github.com/gatsbyjs/gatsby/pull/29447) so if you (or someone else) wants to pick this up again please see both review comments. Thanks!

**Pull request closed without merging**